### DISCUSSION TITLE
How to add multiple assemblies in assembly.js

### DISCUSSION THREAD

**Discussion by GLB-KD** [GMOD/jbrowse-components#1335] [General] [answered]
I am interested in adding multiple assemblies. I have followed the steps as shown in the following link: https://jbrowse.org/jb2/docs/bcc2020_embedding_jbrowse_04_assemblies 

I added one more assembly, however, when I render it I can only see one option in the dropdown. I think I am making some mistake in adding the second assembly details to assembly.js.

**Comment by cmdcolin** [comment]
I would suggest using the full scale jbrowse-web app. This is a little different from the embedding

https://jbrowse.org/jb2/docs/quickstart_web

**Comment by cmdcolin** [comment, marked as answer]
Also congrats on posting the first ever discussion board post :))

I can elaborate more if that answer is too brief. Embedded mode could support multiple assemblies possibly but our full jbrowse 2 aka jbrowse-web app supports this by default, just run multiple `jbrowse add-assembly` CLI commands or manually update the assemblies section of the config.json with multiple assemblies and it will work

**Reply by GLB-KD** [reply]
Thanks, Colin! I was looking for that only.
